media-sound/musikcube/musikcube-0.99.4~git.recipe

```diff
@@ -0,0 +1,130 @@
+SOURCE_DIR_2="asio-$srcGitRev_2"
+PATCHES="musikcube-$portVersion.patchset"
+
+ARCHITECTURES="all ?x86_gcc2"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	musikcube$secondaryArchSuffix = $portVersion
+	cmd:musikcube$secondaryArchSuffix = $portVersion
+	cmd:musikcubed$secondaryArchSuffix = $portVersion
+	lib:libmusikcore$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
+	lib:libavformat$secondaryArchSuffix
+	lib:libavutil$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libev$secondaryArchSuffix
+	lib:libgme$secondaryArchSuffix
+	lib:libmicrohttpd$secondaryArchSuffix
+	lib:libmp3lame$secondaryArchSuffix
+	lib:libmpg123$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	lib:libopenmpt$secondaryArchSuffix
+	lib:libpanelw$secondaryArchSuffix
+	lib:libportaudio$secondaryArchSuffix
+	lib:libswresample$secondaryArchSuffix
+	lib:libtag$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	musikcube${secondaryArchSuffix}_devel = $portVersion
+	devel:libmusikcore$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	musikcube${secondaryArchSuffix} == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libavcodec$secondaryArchSuffix
+	devel:libavformat$secondaryArchSuffix
+	devel:libavutil$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libev$secondaryArchSuffix
+	devel:libgme$secondaryArchSuffix
+	devel:libmicrohttpd$secondaryArchSuffix
+	devel:libmp3lame$secondaryArchSuffix
+	devel:libmpg123$secondaryArchSuffix
+	devel:libncursesw$secondaryArchSuffix
+	devel:libopenmpt$secondaryArchSuffix
+	devel:libpanelw$secondaryArchSuffix
+	devel:libportaudio$secondaryArchSuffix
+	devel:libswresample$secondaryArchSuffix
+	devel:libtag$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	# link submodules
+	rm -rf $sourceDir/src/3rdparty/{asio,bin}
+	ln -s $sourceDir2 $sourceDir/src/3rdparty/asio
+
+	mkdir -p build
+	cd build
+	cmake $cmakeDirArgs .. \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-DCMAKE_INSTALL_LIBDIR="$libDir" \
+		-DCMAKE_INSTALL_PLUGINSDIR="$appsDir/musikcube/plugins" \
+		-DCMAKE_INSTALL_INCLUDEDIR="$includeDir/musikcube/musikcore/sdk" \
+		-DCMAKE_INSTALL_THEMESDIR="$appsDir/musikcube/themes" \
+		-DCMAKE_INSTALL_LOCALESDIR="$appsDir/musikcube/locales" \
+		-DCMAKE_INSTALL_BINDIR="$appsDir/musikcube" \
```

Comment:
Maybe `$appsDir/MusikCube` ... ? (looks more like the Haiku style) ;)

Reply:
Good idea